### PR TITLE
Update README with information on checking for expiring certificates

### PR DESCRIPTION
**Pull request by lgarofalo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,3 +82,18 @@ providing values for the `NEW_ROOTS` and `NEW_INTERMEDIATES` variables:
 ```
 $ NEW_ROOTS="/path/to/root1 /path/to/root2" NEW_INTERMEDIATES="/path/to/int1 /path/to/int22" ./release.sh
 ```
+
+#### Check for expiring roots or intermediates
+
+To verify that an intermediate or root certificate is expiring without creating a release, the `expiring` command can be used from the project root directory.
+
+```
+$ cfssl-trust -d ./cert.db expiring
```
[the remark this answers is not in the document]
Comment (lgarofalo) [reply README.md]: Yea, I didn't notice that either, thanks!

Will update.

[the remark this answers is not in the document]
Comment (kisom) [reply README.md]: Might be useful to document the two parts to the database (intermediates and roots).

[the remark this answers is not in the document]
Comment (lgarofalo) [reply README.md]: @kisom do you just mean referencing that both intermediates and roots are stored in cert.db? Are there any other relevant databases that exist, or is part of the design here that you can run this against any cert db that an individual might have (not the version controlled one)?

[the remark this answers is not in the document]
Comment (kisom) [reply README.md]: The database is a standalone record of the trust store, so it includes both the intermediates and roots. The tool can be run against any cert db someone might have; the one in source control is just the one that we're using internally.

**Pull request merged**